Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -186,21 +186,29 @@ export const differenceBetweenTwoUTC = function (str1) {
 export const storeCallBackForNotification = function (cb) {
   return function (action) {
     return function () {
-      try {
-        const callback = callbackMapper.map(function (notificationType) {
-          if (window.whitelistedNotification.has(notificationType)) {
-            cb(action(notificationType))();
-          }
-        });
-        window.onResumeListeners = [];
-        JBridge.storeCallBackForNotification(callback);
-      }
-      catch (error) {
-        console.log("Error occurred in storeCallBackForNotification ------", error);
+        try {
+         const callback = callbackMapper.map(function (notificationType, notificationBody) {
+            if (window.whitelistedNotification.has(notificationType)) {
+              let parsedBody;
+              try {
+                parsedBody = JSON.parse(notificationBody);
+              } catch (error) {
+                console.error("Failed to parse notificationBody:", error);
+                return;
+              }
+              const { title, msg } = parsedBody;
+              cb(action(notificationType)({title : title, message : msg}))();
```

Comment:
please make it backward-compatible

Reply:
No issue of backward-compatible we are already getting the notificationBody